docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -355,9 +355,11 @@ Consider using the `quarkus.oidc.token.audience` property to verify the token `a
 [[single-page-applications]]
 === Single-page applications
 
+==== Use OIDC provider scripts
```

Comment:
@gsmet this Vale warning seems unwarranted, no?

Reply:
Resolving for the moment